introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #51.            # Change parallel training jobs run by AMT to reduce total training time, constrained by your account limits.
</pre><p>Do we know what the default is? We don't have to note it here necessarily, but maybe it's worth calling out in case users are tempted to try a ridiculous number. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3623/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>Yes, they're explained here https://docs.aws.amazon.com/sagemaker/latest/dg/automatic-model-tuning-limits.html and I will add this link right before the <code>ResourceLimits</code> key.   </p>